cmd/rook/rook/rook.go

```diff
@@ -69,6 +69,7 @@ var (
 //  2. environment variables (upper case, replace - with _, and rook prefix. For example, discovery-url is ROOK_DISCOVERY_URL)
 //  3. command line parameter
 func init() {
+	RootCmd.CompletionOptions.DisableDefaultCmd = true
```

Comment:
it also sets the env variable, is there no harm removing it

Reply:
did not get you, could you exaplain?

Reply:
> 2. environment variables
  3. command line parameter
  
  So I mean it sets the actual  environment variables so that might be used with rook later on to decide whether to use it or not, but not sure